subprojects/docs/src/docs/userguide/upgrading_version_4.adoc

```diff
@@ -17,20 +17,26 @@
 
 This chapter provides the information you need to migrate your older Gradle 4.x builds to Gradle 5.0. In most cases, you will need to apply the changes from all versions that come after the one you're upgrading from. For example, if you're upgrading from Gradle 4.3, you will also need to apply the changes since 4.4, 4.5, etc.
 
-First, a few pointers about upgrading:
+We recommend the following steps for all users:
 
- * Start by running `gradle --scan build` and viewing the https://gradle.com/enterprise/releases/2018.4/#identify-usages-of-deprecated-gradle-functionality[deprecations view] of the generated build scan, or by running `gradle --warn --warning-mode=all build` to see the deprecations in the console.
+ . If you are not already on version 4.10.2, follow the directions <<changes_5.0,below>> to upgrade your project to 4.10.2.
+ . Try running `gradle help --scan` and view the https://gradle.com/enterprise/releases/2018.4/#identify-usages-of-deprecated-gradle-functionality[deprecations view] of the generated build scan.
 +
-This is so that you can see any deprecation warnings that apply to your build. Gradle 5.x will generate (potentially less obvious) errors if you try to upgrade directly to it. 
+image::img/deprecations.png[Deprecations View of a Gradle Build Scan]
```

Comment:
Funny thing about the infrastructure here... `img` is already the basedir for images (https://github.com/gradle/gradle/blob/master/subprojects/docs/docs.gradle#L358), so this won't work. 

You probably want `image::deprecations.png[Deprecations View of a Gradle Build Scan]`